src/main/java/org/trigon/hardware/misc/leds/LEDCommands.java

```diff
@@ -0,0 +1,103 @@
+                    runForLEDs(LEDStrip -> LEDStrip.setCurrentAnimation(() -> LEDStrip.staticColor(color)), LEDStrips);
+                },
+                () -> runForLEDs(LEDStrip::clearLEDColors, LEDStrips),
+                LEDStrips
+        ).ignoringDisable(true);
+    }
+
+    public static Command getBlinkingCommand(Color firstColor, double blinkingIntervalSeconds, LEDStrip... LEDStrips) {
+        return new StartEndCommand(
+                () -> {
+                    runForLEDs((LEDStrip::clearLEDColors), LEDStrips);
+                    runForLEDs((LEDStrip -> LEDStrip.blink(firstColor, blinkingIntervalSeconds)), LEDStrips);
+                    runForLEDs(LEDStrip -> LEDStrip.setCurrentAnimation(() -> LEDStrip.blink(firstColor, blinkingIntervalSeconds)), LEDStrips);
+                },
+                () -> runForLEDs(LEDStrip::clearLEDColors, LEDStrips),
+                LEDStrips
+        ).ignoringDisable(true);
+    }
+
+    public static Command getBreatheCommand(Color color, int amountOfBreathingLEDs, double cycleTimeSeconds, boolean inverted, LarsonAnimation.BounceMode bounceMode, LEDStrip... LEDStrips) {
+        return new StartEndCommand(
+                () -> {
+                    runForLEDs((LEDStrip::clearLEDColors), LEDStrips);
+                    runForLEDs((LEDStrip -> LEDStrip.breathe(color, amountOfBreathingLEDs, cycleTimeSeconds, inverted, bounceMode)), LEDStrips);
+                    runForLEDs(LEDStrip -> LEDStrip.setCurrentAnimation(() -> LEDStrip.breathe(color, amountOfBreathingLEDs, cycleTimeSeconds, inverted, bounceMode)), LEDStrips);
+                },
+                () -> runForLEDs(LEDStrip::clearLEDColors, LEDStrips),
+                LEDStrips
+        ).ignoringDisable(true);
+    }
+
+    public static Command getColorFlowCommand(Color color, double cycleTimeSeconds, boolean inverted, LEDStrip... LEDStrips) {
+        return new StartEndCommand(
+                () -> {
+                    runForLEDs((LEDStrip::clearLEDColors), LEDStrips);
+                    runForLEDs((LEDStrip -> LEDStrip.colorFlow(color, cycleTimeSeconds, inverted)), LEDStrips);
+                    runForLEDs(LEDStrip -> LEDStrip.setCurrentAnimation(() -> LEDStrip.colorFlow(color, cycleTimeSeconds, inverted)), LEDStrips);
+                },
+                () -> runForLEDs(LEDStrip::clearLEDColors, LEDStrips),
+                LEDStrips
+        ).ignoringDisable(true);
+    }
+
+    public static Command getAlternateColorCommand(Color firstColor, Color secondColor, LEDStrip... LEDStrips) {
+        return new StartEndCommand(
+                () -> {
+                    runForLEDs((LEDStrip::clearLEDColors), LEDStrips);
+                    runForLEDs((LEDStrip -> LEDStrip.alternateColor(firstColor, secondColor)), LEDStrips);
+                    runForLEDs(LEDStrip -> LEDStrip.setCurrentAnimation(() -> LEDStrip.alternateColor(firstColor, secondColor)), LEDStrips);
```

Comment:
why call them each time twice and not just make `setCurrentAnimation` call the animation for the first time?